### ISSUE TITLE
Smoother Replay (from Discord: Jeppe)

### ISSUE THREAD

**Issue by jaszlo** [jaszlo/Playerautoma#15]
I will add an option to choose between two types of replays/recording:

* smooth
* choppy

`choppy` is how it is currently. Every game-tick the playerstate is recorded. This results in a "choppy" camera.

`smooth` would record the playerstate every render-tick which should smooth the camera movement

**Comment by jetlagg3d** [comment]
good idea, because currently i'm trying to record parkour in a loop to show as like an example but unfortunately it's innacurate because the camera movements are choppy/late

**Comment by jaszlo** [comment]
@jetlagg3d The replay should be accurate. Or I haven't done to cracy parkour to test it. Could you show me an example where the replay wasn't accurate?
